test-requirements.txt

```diff
@@ -13,7 +16,7 @@ pytest-instafail~=0.3.0
 pytest-mock~=1.1
 pytest-profiling~=1.3.0
 pytest-reorder~=0.1.0
-git+https://github.com/jayvdb/pytest-reqs@skip-vcs-reqs#egg=pytest-reqs
+pytest-reqs~=0.0.6
```

Comment:
undo this line please.

this is why it is breaking, and is a special problem that I need to fix.